cl8-web/tests/unit/specs/__snapshots__/LoginComponent.spec.js.snap

```diff
@@ -14,25 +14,34 @@ exports[`Login.Vue mounting with expected data shows login form when loaded 1`]
   <!---->
   <div>
     <div class="v-mid sign-in-prompt">
-      <h2 class="pt5 tc fw3 tracked">Constellation</h2>
+      <h2 class="pt5 tc fw3 tracked"></h2>
       <div class="w-100 tc">
         <div role="status" aria-live="polite" class="vh dib">
           <!---->
         </div>
         <!---->
         <form class="w-100 pa3 dib border-box mw6 ph5">
-          <p class="gray measure tl lh-copy">Please enter the email address that is associated with your Icebreaker One CL8 account. We will send a one time login code, to finish logging you in:</p>
-          <div class="w-100 mb3"><input type="text" name="email" placeholder="your email address" aria-label="your email address" autocomplete="email" class="input-reset br2 pa2 ba b--light-gray mt1 w-100">
+          <p class="gray measure tl lh-copy">
```

Comment:
I think unfortunate side effect of the mocking is that we now show _nothing_ in these snapshots. This reduces their utility, but I'm not aware of us using them anyway for now.

@rmparr Ryan can you please create an issue to research this in future - we'd either get rid of snapshots entirely from the repo, or we'd make an explicit decision about how we use them.